packages/coordinator/src/components/Popup/NotificationPopup.tsx

```diff
@@ -0,0 +1,58 @@
+import { Dialog } from "@material-ui/core";
+import React from "react";
+import styles from "./Popup.module.scss";
+import DialogActions from "@material-ui/core/DialogActions";
+import Button from "../Button";
+import Text from "../Text";
+
+type PopupProps = {
+  buttonApproveText: string;
+  open: boolean;
+  titleFirst: string;
+  titleSecond?: string;
+  className?: string;
+  onClose: () => void;
+  PopupButton?: React.ReactNode;
+};
+
+export function NotificationPopup({
```

Comment:
Don't we already have a popup component in coordinator?

Reply:
Popup split into three: 

1. DialogBase skeleton component
2. Popup implementation
3. NotificationPopup implementation